akka-persistence-query/src/main/scala/akka/persistence/query/scaladsl/DurableStateStoreQuery.scala

```diff
@@ -26,8 +26,8 @@ trait DurableStateStoreQuery[A] extends DurableStateStore[A] {
    * This will return changes that occurred up to when the `Source` returned by this call is materialized. Changes to
    * objects made since materialization are not guaranteed to be included in the results.
    *
-   * The [[DurableStateChange]] elements can be [[akka.persistence.query.UpdatedDurableState]] or `DeletedDurableState`.
-   * `DeletedDurableState` is not implemented yet, see issue https://github.com/akka/akka/issues/30446.
```

Comment:
search for `30446` and you will find a few more similar comments

Reply:
there is also "`DeletedDurableState` is not implemented yet" in akka-docs/src/main/paradox/durable-state/persistence-query.md

it would also be good to mention the new effect in akka-docs/src/main/paradox/typed/durable-state/persistence.md line 155...